mrtt-ui/src/styles/forms.js

```diff
@@ -1,16 +1,29 @@
+import { Typography } from '@mui/material'
 import { styled } from '@mui/material/styles'
 
-export const MainFormDiv = styled('div')(() => ({
+export const MainFormDiv = styled('div')`
   display: 'flex',
-  flexDirection: 'column',
-  justifyContent: 'center',
-  alignItems: 'center',
-  margin: '1.5em'
-}))
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  padding: 1.5em;
+  width: 100%;
+  box-sizing: border-box;
+`
 
 export const FormQuestionDiv = styled('div')(() => ({
   display: 'flex',
   flexDirection: 'column',
   marginBottom: '1em',
   marginTop: '2em'
 }))
+
+export const SectionFormTitle = styled(Typography)`
+  marginbottom: '0.5em';
+`
+SectionFormTitle.defaultProps = { variant: 'h4' }
+
+export const Form = styled('form')`
+  display: flex;
+  flex-direction: column;
+`
```

Comment:
Also I think for the alignment of the error text. I cant remember, but this is all editable in your or my code.